webpack.config.js

```diff
@@ -63,7 +63,8 @@ var webpackConfig = {
     "lockup":               path.resolve(__dirname, srcSass + "/components/lockup.component.scss"),
     "logo":                 path.resolve(__dirname, srcSass + "/components/logo.component.scss"),
     "media":                path.resolve(__dirname, srcSass + "/components/media.component.scss"),
-    "quote":                path.resolve(__dirname, srcSass + "/components/quote.component.scss")
+    "quote":                path.resolve(__dirname, srcSass + "/components/quote.component.scss"),
+    "layouts":              path.resolve(__dirname, srcSass + "/layouts/index.scss")
```

Comment:
Hi @sherakama,

This is a start to get things working.

Two questions for you:
1) Sanity check: Does my approach make sense? Does putting this work in the layout make sense? 

2) I'm missing a step to get this to the browser. Can you see what it is I am missing?
It creates a file in /dist, but that file doesn't show up on the page in the browser.